### DISCUSSION TITLE
Multizone mesh CGNS

### DISCUSSION THREAD

**Discussion by Gittwell** [su2code/SU2#2358] [Q&A]
Hello. I get an error when run SU2_DEF.
How can I change multizone mesh to one-zone mesh. Or fix this problems?

Reading the CGNS file: wing.cgns.
WARNING: The CGNS file version (3.11)  is old and may cause high memory usage issues, consider updating the file with the cgnsupdate tool.
CGNS file contains 1 database(s).
Database 1, BASE#1:  cell dimension of 2, physical dimension of 3.
9 total zone(s).

Error in "void CCGNSMeshReaderFVM::ReadCGNSZoneMetadata()":
-------------------------------------------------------------------------
CGNS reader currently expects only 1 zone per CGNS file.Multizone problems can be run with separate CGNS files for each zone.

**Comment by pcarruscag** [comment]
You can separate your mesh into multiple zones and use each of the new meshes in one zone.

**Reply by Gittwell** [reply]
The approach as described here?
https://su2code.github.io/docs_v7/Multizone/

With what software can i split the mesh? Does su2 have utilities for this?

**Reply by pcarruscag** [reply]
If it can't read a multizone mesh how would it split it? You have to use something else.
